packages/app/package.json

```diff
@@ -28,6 +28,7 @@
     "@backstage/plugin-newrelic": "^0.2.6",
     "@backstage/plugin-org": "^0.3.12",
     "@backstage/plugin-pagerduty": "0.3.2",
+    "@backstage/plugin-github-release-manager": "^0.1.1",
```

Comment:
Skip this if it's not being added to the app right, or should it?

Reply:
> Skip this if it's not being added to the app right, or should it?

I believe this was part of the `create plugin` bootstrap script, so simply didn't remove it 🤷‍♂️ 

What does this dependency mean in practical terms?

Reply:
Decided to remove it until I understand what it means 🙂